docs/sources/flow/reference/components/prometheus.exporter.windows.md

```diff
@@ -29,9 +29,10 @@ prometheus.exporter.windows "LABEL" {
 The following arguments can be used to configure the exporter's behavior.
 All arguments are optional. Omitted fields take their default values.
 
-| Name                       | Type           | Description                                                   | Default | Required |
-|----------------------------|----------------|---------------------------------------------------------------|---------|----------|
-| `enabled_collectors`       | string`        | List of collectors to enable.                                |         | no       |
+| Name                 | Type       | Description                               | Default | Required |
+|----------------------|------------|-------------------------------------------|---------|----------|
+| `enabled_collectors` | `string`   | List of collectors to enable.             |         | no       |
+| `timeout`            | `duration` | Configure timeout for collecting metrics. | `4m`    | no       |
```

Comment:
This new timeout gives me a bit of pause, this seems an artifact from windows_exporter running standalone but generally the scrape_timeout will be the driver. Unfortunately there isn't a way to propagate that around. Thoughts on changing the doc to something like `Configure timeout for windows_exporter serving metrics, this does not override any scraping timeouts`, though dang that is wordy. The other path would be to make it hard coded to the default and not expose it for the time being. Any thoughts?

Reply:
Good call, not in love with the verbiage around it but cant think of anything better and we can always add context if we get questions.

Reply:
Okay, then I would reverted timeout change here to the hardcoded one...  adding the context should do in a separate PR?